src/core/k8.ts

```diff
@@ -1256,11 +1337,20 @@ export class K8 {
     return body as k8s.V1Status
   }
 
-  private _handleKubernetesClientError (response: http.IncomingMessage, error: Error | any, errorMessage: string) {
-    const statusCode = +response.statusCode
+  /* ------------- Utilities ------------- */
+
+  /**
+   * @param response - response object from the kubeclient call
+   * @param error - body of the response becomes the error if the status is not OK
+   * @param errorMessage - the error message to be passed in case it fails
+   *
+   * @throws SoloError - if the status code is not OK
+   */
+  private _handleKubernetesClientError (response: http.IncomingMessage, error: Error | any, errorMessage: string): void {
```

Comment:
```suggestion
  private handleKubernetesClientError (response: http.IncomingMessage, error: Error | any, errorMessage: string): void {
```
https://google.github.io/styleguide/tsguide.html#identifiers-underscore-prefix-suffix